Classes/Middleware/TypoScriptRenderingMiddleware.php

```diff
@@ -73,8 +73,12 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
      */
     private function amendContentType(ResponseInterface $response, string $requestedContentType): ResponseInterface
     {
-        $originalContentTypeHeader = $response->getHeader('Content-Type')[0];
-        if (strpos($originalContentTypeHeader, self::defaultContentType) === 0 && strpos($originalContentTypeHeader, $requestedContentType) === false) {
+        $originalContentTypeHeader = $response->getHeader('Content-Type')[0] ?? '';
+        if (
+            $originalContentTypeHeader === ''
```

Comment:
I wonder if having the replaced content header makes sense if the original header is empty …

Possibly ditching the `=== ''` check and relying on the other two checks might suffice.

@helhum What do you think?

Reply:
works without this check as well.
i updated the issue with information, how this happens.